API/config.example.json

```diff
@@ -1,14 +1,14 @@
 {
-    "server": {
-      "port": 3000
-    },
-    "database": {
-      "host": "localhost",
-      "port": 5432,
-      "username": "",
-      "password": "",
-      "name": "risk",
-      "dialect": "pg",
-      "minconnections": 2
-    }
-}
\ No newline at end of file
+  "server": {
+    "port": 3000
+  },
+  "database": {
+    "host": "localhost",
+    "port": 5432,
+    "username": "postgres",
+    "password": "password",
+    "name": "risk",
+    "dialect": "pg",
+    "minconnections": 2
```

Comment:
This file was meant to be an example.  Ideally, you'd copy it to a new file called config.json.  That config should be then added to your gitignore.  Basically, it keeps your config file local.  This is because it's bad practice to publish your database password on the internet.